torch/_inductor/kernel/mm_plus_mm.py

```diff
@@ -151,7 +151,7 @@ def tuned_mm_plus_mm(mat1, mat2, mat3, mat4, *, layout=None):
     ):
         # TODO(jansel): support different K values when this is fixed:
         # https://github.com/openai/triton/issues/967
-        return lowerings[aten.addmm](lowerings[aten.mm](mat1, mat2), mat3, mat4)
+        return lowerings[aten.addmm](lowerings[aten.mm](mat3, mat4), mat1, mat2)
```

Comment:
this doesn't seem like a correct fix, as arguments could come in a different order, and mm(mat1, mat2) would be a small matrix that would need to be broadcasted (and will fail to be broadcasted) to mm(mat3, mat4). In this case we need to return the original expression, mm + mm, and not use addmm (there's a similar fix in a pattern that matches add + mm  to addmm, sometimes it's impossible)

Reply:
Yes, the original is not a correct fix, please see the latest code.